.vscode/launch.json

```diff
@@ -0,0 +1,17 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "type": "pwa-node",
+            "request": "launch",
+            "name": "Launch Program",
+            "skipFiles": [
+                "<node_internals>/**"
+            ],
+            "program": "${workspaceFolder}\\app.js"
+        }
+    ]
+}
```

Comment:
If we want to use nodemon while debugging which will have an automatic restart upon code changes
We can use this template
![image](https://user-images.githubusercontent.com/25090936/109425936-ae8a6b00-799f-11eb-88a5-b2d42f8dd2b0.png)